kit/score/registry.go

```diff
@@ -69,7 +73,12 @@ func (s *registry) Add(test interface{}, taskName string, max, weight int) {
 // conjunction with MaxByName and MinByName called from within a subtest.
 //
 // Will panic if the test has already been registered or if max or weight is non-positive.
-func (s *registry) AddSub(test interface{}, taskName, subTestName string, max, weight int) {
+func (s *registry) AddSub(test interface{}, subTestName string, max, weight int) {
+	tstName := fmt.Sprintf("%s/%s", testName(test), subTestName)
+	s.add(tstName, "", max, weight)
+}
+
+func (s *registry) AddSubWithTaskName(test interface{}, taskName, subTestName string, max, weight int) {
```

Comment:
Add doc comment.

Suggest to swap `taskName` and `subTestName`, since the first argument is the main test "name".